classes/rental.rb

```diff
@@ -8,26 +8,57 @@ class Rental
   def initialize(date, book, person)
     @date = date
     @book = book
+    book.rentals << self
     @person = person
+    person.rentals << self
```

Comment:
- [ ] Again there is an error in creating and saving a rental.  It seems like you are trying to add the current rental (self) to the rentals collection of a Book object (book). However, the book object is nil, which means it has not been instantiated properly.
To fix this issue, ensure that you properly create and initialize the book object before using it. You should make sure that you have an instance of the Book class and that it is correctly associated with the Rental class.
 
e.g

```
class Rental
  attr_accessor :date, :book, :person

  def initialize(date, book, person)
    @date = date
    @book = book
    @person = person

    # Make sure to initialize the rentals array for the associated book.
    @book.rentals ||= []
    @book.rentals << self
  end
end
```


Screenshot
![image](https://github.com/Kel-nana/school_library/assets/81932478/b63cfefd-c60b-4045-ba9b-a9251fb693e8)